### ISSUE TITLE
Investigate shared authentication between host and plugins

### ISSUE THREAD

**Issue by wtrocki** [aerogear/charmil#70]
How plugins can donate authentication and how host can authenticate all plugins if needed.

**Comment by wtrocki** [comment]
@namit-chandwani Please do not assign yourself to issues that you do not work on right now. 
You cannot work on more than one issue at the time right ?

**Comment by namit-chandwani** [comment]
> @namit-chandwani Please do not assign yourself to issues that you do not work on right now.
> You cannot work on more than one issue at the time right ?

Actually, I had researched a bit on this issue last week. But then I switched to the config storage one, once I realized that it had to be resolved before this.

But yes you're right, I should be assigning myself to only one issue at a time. 
Will surely keep this in mind in the future :)

**Comment by wtrocki** [comment]
Yes.. you have 2 issues in progress. This would be waiting for you as next task. 
We have project where we can keep the backlog updated 

https://github.com/aerogear/charmil/projects

**Comment by namit-chandwani** [comment]
As discussed on chat yesterday, this feature is out of scope currently.
Moving this to Todo once again 😅

**Comment by wtrocki** [comment]
@ankithans @namit-chandwani 

I need to do some reaserch myself. Assigning
